compose.yaml

```diff
@@ -1,8 +1,64 @@
 services:
   # Controller network
+  premiscale-single-host:
+    container_name: premiscale-single-host
+    platform: linux/amd64
+    profiles:
+      - single_host
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+      target: develop
+      pull: true
+    environment:
+      - PREMISCALE_TOKEN=123
+      - PREMISCALE_CONFIG_PATH=/opt/premiscale/config.yaml
+      - PREMISCALE_LOG_LEVEL=debug
+      - PREMISCALE_PLATFORM=platform:443
+      - PREMISCALE_CACERT=/opt/premiscale/cert.pem
+      - INFLUXDB_HOST=influxdb:8086
+      - INFLUXDB_USERNAME=premiscale
+      - INFLUXDB_PASSWORD=premiscale
+      - MYSQL_HOST=mysql-agent-state:3306
+      - MYSQL_USERNAME=premiscale
+      - MYSQL_PASSWORD=premiscale
+    networks:
+      - controller
+    deploy:
+      resources:
+        limits:
+          cpus: '4.0'
+          memory: 1G
+        reservations:
+          cpus: '1.0'
+          memory: 1G
+    depends_on:
+      mysql-agent-state:
+        condition: service_healthy
+      influxdb:
+        condition: service_healthy
+      echoes:
+        condition: service_healthy
+      registration:
+        condition: service_healthy
+      platform:
+        condition: service_healthy
+    volumes:
+      - ./certs/cert.pem:/opt/premiscale/cert.pem
+      - ./src/tests/data/config/v1alpha1_base+qemu_single_host_no_asg.yaml:/opt/premiscale/config.yaml
```

Comment:
Main difference is, we can now launch a local compose-based stack with test configs and select which configuration to launch based on the compose profile.

Reply:
Really looking forward to getting to a place where I can launch a local minikube cluster, same as with pass-operator, but I can launch with different devspace profiles and edit particular fields, etc. Maybe in the next several weeks or months.